src/main/resources/com/puppycrawl/tools/checkstyle/checks/sizes/messages.properties

```diff
@@ -2,7 +2,7 @@ executableStatementCount=Executable statement count is {0,number,integer} (max a
 maxLen.anonInner=Anonymous inner class length is {0,number,integer} lines (max allowed is {1,number,integer}).
 maxLen.file=File length is {0,number,integer} lines (max allowed is {1,number,integer}).
 maxLen.lambdaBody=Lambda body length is {0,number,integer} lines (max allowed is {1,number,integer}).
-maxLen.method=Method length is {0,number,integer} lines (max allowed is {1,number,integer}).
+maxLen.method=Method {2} length is {0,number,integer} lines (max allowed is {1,number,integer}).
```

Comment:
@romani What is your opinion on our of order parameters?

Reply:
Note that we did not do an extensive search on the ordering of the parameters relative to the string markers. We assumed that this would not matter given that some languages would need different orderings. For example, the Chinese translations have the {1} and {0} parameters in a different order than the English (original) order. If you would like the English to be in {0}, {1}, {2} order, we would need to change the order of the args passed to log() and the numbering of all the parameters on all the all the translation strings instead of just adding the {2}.

Reply:
Let's keep it as it is now.
Refactoring of other languages might be error prone